libs/common/src/lib/localization/translation.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { tx, t, normalizeLocale } from "@transifex/native";
+import { ConfigService } from "@nestjs/config";
+
+@Injectable()
+export class TranslationService {
+  constructor(private readonly configService: ConfigService,) {
```

Comment:
If the service isn't going to be used after the class is constructed, there's no reason to make it a property of the class, so this can be simpler:
```
constructor(configService: ConfigService) {
```